app/models/manageiq/providers/vmware/cloud_manager/orchestration_service_option_converter.rb

```diff
@@ -3,77 +3,122 @@ class Vmware::CloudManager::OrchestrationServiceOptionConverter < ::ServiceOrche
     include Vmdb::Logging
 
     def stack_create_options
+      @template = ManageIQ::Providers::Vmware::CloudManager::OvfTemplate.new(self.class.get_template(@dialog_options).content)
```

Comment:
Prefer this just be a local variable since the only users look like they are directly called from this method, you can just pass it as an argument and keep the instance vars down